config/config.go

```diff
@@ -165,7 +164,7 @@ type SmeshingConfig struct {
 func DefaultConfig() Config {
 	return Config{
 		BaseConfig:      defaultBaseConfig(),
-		Genesis:         DefaultGenesisConfig(),
+		Genesis:         *DefaultGenesisConfig(),
```

Comment:
Why not also change `DefaultGenesisConfig` from returning a value instead of a pointer? You changed `DefaultTestGenesisConfig` as well?

Reply:
I forgot